01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,186 @@ using InteractiveUtils
+It is evident that the more we know about the reality in which we live, the more we will be able to modify it. 
+But from this idea comes a very important question: how is the "order" in which innovation takes place? 
+Is it necessary first to have an absolute understanding of the process we are trying to modify? 
+Or is technology created in a more chaotic process, through trial and error?
 
+So let´s think for a moment, does a child need to know the gear mechanisms of a bicycle, in order to be able to use it? does Messi need to know about fluid dynamics to make the ball take a curved trajectory? Did the Romans need to know the Navier-Stokes equation in order to build their huge aqueducts? 
+Maybe you are thinking that these are very specific cases, but the reality is that those example share a general and universal property: The knowledge is acquired based on experimentation, implementation and heuristics, pure trial and error. 
+And technological innovation follows the same mechanism, it was discovered by people that were  taking risks, taking action, with a pure trial-and-error experiential manner and, above all, without anyone truly understanding the theory behind.
 
-If you start looking at the technology/innovation history you may gradually begin to notice a repeating pattern: There is a constant differentiation between the people who *do* and the people who *watch* and then narrate. "The History Written by the Losers" would [Nassim Nicholas Taleb](https://en.wikipedia.org/wiki/Nassim_Nicholas_Taleb) said, the people that are doing stuff don't have time for beauty writing. But the thing does not stop there. It is not only that the do-nothings who write about the findings of others, but also attribute it to them! And this comes from a worrying bias that has been developing throughout recent history: the linear thinking we are talking about. That –apparently– common sense in which knowledge is built from a purely intellectual work that can be done in the armchair at home, and that after acquiring this sacred theoretical knowledge it is possible to come up with technology or innovation.
+The current understanding of how knowledge is produced is quite out of step with what really happen, and may even work against us in our development. 
+Nowadays, there is a kind of *linear thinking* model in which is taken for granted that the process of knowledge production is very definite: First, academic and theoretical discoveries are made and only then, with all of that formalization achieve, one can start thinking of putting it to practical use in the real world. 
```

Comment:
Don't use italics.

Reply:
There are some gramatical mistakes in this sentence (**it** is taken for granted, achieve)
I would rework the whole sentence and try to use less words, also try to avoid using the word definite in this context

Reply:
Changed by:
Nowadays, there is a kind of linear thinking model in which it is taken for granted that the process of knowledge production has a very specific way of occurring: First, academic and theoretical discoveries are made. And only then, with all of that formalization achieve, one can start thinking of putting it into practical use.
This type of thinking is counterproductive because, being so widespread, it causes many people to be more concerned with constantly acquiring theoretical knowledge, rather than taking action and immersing themselves in practice.